src/bindings.ts

```diff
@@ -25,6 +25,7 @@ export interface Bindings {
 	// Key rotation schedule
 	ROTATION_CRON_STRING?: string;
 	KEY_LIFESPAN_IN_MS: string;
+	KEY_NOT_BEFORE_DELAY_IN_MS: string;
```

Comment:
We are measuring in ms because lifespan is already measured in ms, correct? Since we always use hours for these it feels seconds or minutes would be a precise enough unit.

Reply:
yes. I did not want to have multiple unit, even though that means one more conversion